package.json

```diff
@@ -73,6 +73,7 @@
     "@sentry/react": "^5.22.3",
     "@sentry/tracing": "^5.22.3",
     "@sindresorhus/slugify": "1.0.0",
+    "@taktikal/fillandsign": "^3.1.0",
```

Comment:
Actually I think there is a preference to pin down the versions of the packages as the one above.

Reply:
We should add a yarn config file that enforces this as a default.